### DISCUSSION TITLE
Wind direction with respect to north

### DISCUSSION THREAD

**Discussion by santiagolarregla** [PaddlePaddle/PaddleSpatial#160] [BaiduKDDCup2022] [answered]
Is Ndir the angle between the nacelle and the North direction?
If this is so, and understanding that Wdir is the angle of the wind direction with respect to the nacelle, then the angle of the wind with North would be Ndir+Wdir, is this correct? 
This is a picture of what I have understood of the variables, but the definition of them is very confusing, could you confirm that this is so?
Thank you in advance.
<img width="887" alt="foto_viento" src="https://user-images.githubusercontent.com/105053980/172662237-49543d05-74a1-4954-aac0-443331c7b558.png">

**Comment by ethanyxx** [comment, marked as answer]
Hi,

Thanks for the questions! The interpretation you presented is correct.

Best,
